tensorflow/stream_executor/rocm/roctracer_wrapper.h

```diff
@@ -0,0 +1,90 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
```suggestion
/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Reply:
this will be fixed in the next commit/push